public/js/utils.jsx

```diff
@@ -41,6 +41,11 @@ export function gettext(string){
 }
 
 
+export function setTitle(string) {
+  document.title = gettext('Pay') + ' | ' + string;
```

Comment:
Marketplace does the pipe but does it as a suffix. Personally I'm not tied to it, blah lets remove it.